.github/workflows/ci.yml

```diff
@@ -111,6 +110,13 @@ jobs:
         with:
           submodules: true
           fetch-depth: 0
+      - name: Build GeoIP
+        run: |
+          git clone --depth 1 https://github.com/maxmind/geoip-api-c.git
```

Comment:
```suggestion
          git clone --depth 1 --no-checkout https://github.com/maxmind/geoip-api-c.git
          git fetch --tags
          # Check out the last release, v1.6.12
          git checkout 4b526e7331ca1d692b74a0509ddcc725622ed31a
```

Reply:
I think this wold be more clear and we don't need to follow the releases:
```
git clone https://github.com/maxmind/geoip-api-c.git
cd geoip-api-c/
latest=$(git describe --tag | cut -d"-" -f1)
git checkout tags/$latest
```
Now `git status` shows me
```
$ git status
HEAD detached at v1.6.12
nothing to commit, working tree clean
```
What do you think?

Reply:
I wanted to pin the commit hash, a tag can be modified. You want to be sure that you always build with the same version. The alternative would be to download the release and verify the checksum. Also, this is the last release. The repository was archived and no new releases will be published anyway, so you will never have to update the version anyway.

Reply:
> Also, this is the last release. The repository was archived and no new releases will be published anyway, so you will never have to update the version anyway.

Ah, you're right, this make sense - thanks.

Btw I think the `--no-checkout` option and `git fetch tags` (and `--depth 1`) aren't necessary.

This would be enough:
```
git clone https://github.com/maxmind/geoip-api-c.git
git checkout 4b526e73
```
Or why are they necessary there?

Reply:
That is true, but then you clone the full history. `--no-checkout` is just to save some CPU cycles.

Reply:
Thanks, I added your suggestion.